pkg/common/config/config.go

```diff
@@ -293,16 +293,15 @@ func validateConfig(ctx context.Context, cfg *Config) error {
 			vcConfig.InsecureFlag = cfg.Global.InsecureFlag
 		}
 	}
-	clusterFlavor, err := GetClusterFlavor(ctx)
-	if err != nil {
-		return err
-	}
+
 	if cfg.NetPermissions == nil {
 		// If no net permissions are given, assume default
 		log.Info("No Net Permissions given in Config. Using default permissions.")
-		if clusterFlavor == cnstypes.CnsClusterFlavorVanilla {
-			cfg.NetPermissions = map[string]*NetPermissionConfig{"#": GetDefaultNetPermission()}
-		}
+		// TODO:
```

Comment:
Can we track this with a task or an issue in GitHub so that we do not forget to revert this change?

Reply:
+1